### PR TITLE
feat: use Ace's Flix mode

### PR DESCRIPTION
**Pull request by sockmaster27**

### CODE REVIEW

File: src/util/InlineEditor.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React, {Component} from 'react';
 import AceEditor from 'react-ace'
-import 'brace/mode/scala'
 
-import FlixMode from './FlixMode'
-import 'brace/theme/chrome';
+import 'ace-builds/src-noconflict/mode-flix'
+import 'ace-builds/src-noconflict/theme-chrome'
```
Comment (magnus-madsen) [inline src/util/InlineEditor.js:6]: Can this entire class be dropped?

**Comment by magnus-madsen** [comment]
Tested, no breakages?

File: src/util/InlineEditor.js (hunk 1 above)
Comment (sockmaster27) [reply src/util/InlineEditor.js:6]: I believe so. It's strictly about highlighting, right?

**Comment by sockmaster27** [comment]
Only what I already wrote relating to play.flix.dev

File: src/util/InlineEditor.js (hunk 1 above)
Comment (magnus-madsen) [reply src/util/InlineEditor.js:6]: Yeah.

Comment (magnus-madsen) [reply src/util/InlineEditor.js:6]: Can do in follow up PR

Comment (sockmaster27) [reply src/util/InlineEditor.js:6]: If we're both talking about `FlixMode` then it's already removed :)

Comment (magnus-madsen) [reply src/util/InlineEditor.js:6]: I meant InlineEditor?

Comment (magnus-madsen) [reply src/util/InlineEditor.js:6]: Or maybe it is needed as a separate component?

Comment (sockmaster27) [reply src/util/InlineEditor.js:6]: Ah, right now `InlineEditor` is used for the uneditable code boxes

Comment (magnus-madsen) [reply src/util/InlineEditor.js:6]: OK

**Pull request merged**